templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -1,11 +1,15 @@
 const initNavigationSliding = () => {
-  const navigation = document.querySelector('.p-navigation--sliding');
-  const toggles = document.querySelectorAll('.p-navigation__link[aria-controls]:not(.js-back)');
+  const navigation = document.querySelector('.p-navigation--sliding, .p-navigation--reduced');
+  const secondaryNavigation = document.querySelector('.p-navigation--reduced + .p-navigation');
+  const toggles = document.querySelectorAll('.p-navigation__nav .p-navigation__link[aria-controls]:not(.js-back)');
   const searchButtons = document.querySelectorAll('.js-search-button');
   const menuButton = document.querySelector('.js-menu-button');
   const dropdowns = document.querySelectorAll('ul.p-navigation__dropdown');
-  const mainList = dropdowns[0].parentNode.parentNode;
-  const lists = [...dropdowns, mainList];
+  const lists = [...dropdowns];
+  const mainList = dropdowns[0] ? dropdowns[0].parentNode.parentNode : null;
```

Comment:
Nit: why not use optional chaining instead of a ternary? 

```suggestion
const mainList = dropdowns?.[0]?.parentNode.parentNode;
```
This assumes each element always has a parent node and that parent node has a parent node, as your code does; but checks for existence of first dropdown element at same time as assignment. However, mainList would be `undefined` instead of `null` in the case that the first element doesn't exist.

Reply:
> Nit: why not use optional chaining instead of a ternary?

Mostly because when I first learned JS we didn't have such shiny toys :D 

But, of course, this is a very good suggestion. Will address.